lib/configuration.js

```diff
@@ -44,6 +44,44 @@ function applyLoggingOptions(options) {
 }
 
 
+function deprecate(config = {}) {
+  if (config.options) {
+    if (config.options.c) {
+      throw new Error('DEPRECATED: Dredd does not support '
```

Comment:
Since this is erroring, this is not deprecation. It's removal. Deprecation would be when it still works but is discouraged and potentially scheduled for removal in the future:

> In several fields, deprecation is the discouragement of use of some terminology, feature, design, or practice, typically because it has been superseded or is no longer considered efficient or safe, without completely removing it or prohibiting its use. It can also imply that a feature, design, or practice will be removed or discontinued entirely in the future

From [Wikipedia on Deprecation](https://en.wikipedia.org/wiki/Deprecation).

Reply:
To fix, if you want to keep this behaviour I think you can rename it to now include "deprecated". Alternatively you could actually deprecate it and set `config.options.equivilent option = foo` etc

Reply:
You're right. But while I could translate some of the options to the new ones, certain features are just gone. What word should I use to alert the user about the fact the feature is gone? If I don't do this, `optimist` silently passes even with extra options (yeah, facepalm).

I want Dredd to fail fast in case I know someone just tried to use a feature which is not in place anymore.

Reply:
> I think you can rename it to now include "deprecated"

Took me a moment to parse the sentence, but if you meant

> I think you can rename it to **not** include "deprecated"

then I could do that, yes 😄

Reply:
072ce78

Reply:
ff55694